string modelName
geometry_msgs/Vector3 modelPosition
geometry_msgs/Quaternion modelOrientation

